### ISSUE TITLE
Resubmitting the AS3 declaration for TS profiles throws eerror

### ISSUE THREAD

**Issue by focrensh** [F5Networks/f5-appsvcs-extension#370]
### Environment
 * 3.22.1
 * 14.1

### Summary
The declaration works on each call, but any call after the initial one will return the error below. The declaration does appear to be successful in its changes though.

```"01070344:3: Cannot delete referenced virtual address /Common/Shared/255.255.255.254.",```

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "AS3",
    "action": "deploy",
    "persist": true,
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.22.0",
        "remark": "Bacon",
        "Common": {
            "Shared": {
                "class": "Application",
                "template": "shared",
                "telemetry_local_rule": {
                    "class": "iRule",
                    "iRule": "when CLIENT_ACCEPTED {\n  node 127.0.0.1 6514\n}"
                },
                "telemetry_local": {
                    "class": "Service_TCP",
                    "virtualAddresses": [
                        "255.255.255.254"
                    ],
                    "virtualPort": 6514,
                    "iRules": [
                        "telemetry_local_rule"
                    ]
                },
                "telemetry": {
                    "class": "Pool",
                    "members": [
                        {
                            "enable": true,
                            "serverAddresses": [
                                "255.255.255.254"
                            ],
                            "servicePort": 6514
                        }
                    ],
                    "monitors": [
                        {
                            "bigip": "/Common/tcp"
                        }
                    ]
                },
                "telemetry_hsl": {
                    "class": "Log_Destination",
                    "type": "remote-high-speed-log",
                    "protocol": "tcp",
                    "pool": {
                        "use": "telemetry"
                    }
                },
                "telemetry_formatted": {
                    "class": "Log_Destination",
                    "type": "splunk",
                    "forwardTo": {
                        "use": "telemetry_hsl"
                    }
                },
                "telemetry_publisher": {
                    "class": "Log_Publisher",
                    "destinations": [
                        {
                            "use": "telemetry_formatted"
                        }
                    ]
                },
                "telemetry_security_log_profile": {
                    "class": "Security_Log_Profile",
                    "application": {
                        "localStorage": false,
                        "remoteStorage": "splunk",
                        "protocol": "tcp",
                        "servers": [
                            {
                                "address": "255.255.255.254",
                                "port": "6514"
                            }
                        ],
                        "storageFilter": {
                            "requestType": "all"
                        }
                    }
                }
            }
        }
    }
  }
```

2. Observe the following error response:
```json
        {
            "code": 200,
            "message": "success",
            "lineCount": 16,
            "host": "localhost",
            "tenant": "Common",
            "runTime": 6313
        },
        {
            "code": 422,
            "message": "declaration failed",
            "response": "01070344:3: Cannot delete referenced virtual address /Common/Shared/255.255.255.254.",
            "host": "localhost",
            "tenant": "Common",
            "runTime": 4106
        }
```

### Expected Behavior
"no change" returned

### Actual Behavior
Error above.

**Comment by dstokesf5** [comment]
I have confirmed the bug and added it to our internal product backlog as AUTOTOOL-2035.

I noticed your Common tenant is missing the class property. While the schema allows this, it causes a lot of strange behavior. I am still able to reproduce this bug when this property is present.

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 3.25.0 release.
